apps/communications/dialer/test/marionette/add_contact_from_dialer.js

```diff
@@ -0,0 +1,38 @@
+/* global require, marionette, setup, suite, test */
+'use strict';
+
+var assert = require('chai').assert;
+
+var Dialer = require('./lib/dialer.js');
+
+marionette('Dialer > Keypad', function() {
+
+  var client = marionette.client({
+    desiredCapabilities: { raisesAccessibilityExceptions: false }
+  });
+
+  var dialerApp;
+
+  setup(function() {
+  
+    dialerApp = new Dialer(client);
+    dialerApp.launch();
+  
+   });
+
+  suite('Test Suite', function() {
+
+    test('Add a contact from dialer', function() {
```

Comment:
This was a question for this afternoon meeting..if I should create more test for this suite or include it in another one as it is a very short test...I will try to add it to  that keypad_test.js